unfoldingWord/translationCore
# Feature #3273 /Update to use UGNT resources

#### This pull request addresses:

Update app to use UGNT resources.


##### Merge order: 
ScripturePane: https://github.com/translationCoreApps/ScripturePane/pull/82
translationCore: https://github.com/unfoldingWord-dev/translationCore/pull/3639
usfm-js: https://github.com/translationCoreApps/usfm-js/pull/26
tC_Resources: https://github.com/translationCoreApps/tC_Resources/pull/15

#### How to test this pull request:

Need these parts loaded:
ScripturePane: https://github.com/translationCoreApps/ScripturePane/pull/82
tC_Resources: https://github.com/translationCoreApps/tC_Resources/pull/15

Import a new project from USFM.  When you open the WA and TW tools, they will initialize alignment using UGNT and should work fine (greek words shown).  Should see UGNT as option for source. Add it as a source language and compare to BHP.  All the words should be there.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3639)
